There is no Spoon - Episode 1/en345/There_is_no_spoon_Episode_1.cs

```diff
@@ -1,8 +1,6 @@
 // # A program that resolves the "There is no spoon - Episode 1" problem from [Coding game website](https://www.codingame.com/home)
-
 // ## The problem and the solution
```

Comment:
Explain what the problem is. There are no details here!

Reply:
The details about the problem and the approach used in the solution were part of the image included in the page. Isn't that good enough also?